Tests/BackendIntegrationTests/OfflineStoreKitIntegrationTests.swift

```diff
@@ -241,16 +243,23 @@ class OfflineStoreKit1IntegrationTests: BaseOfflineStoreKitIntegrationTests {
 
     @available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *)
     func testCallToGetCustomerInfoWithPendingTransactionsPostsReceiptOnlyOnce() async throws {
+        // forceRenewalOfSubscription doesn't work well, so we use this instead
+        if #available(iOS 16.4, *) {
+            self.testSession.timeRate = .oneRenewalEveryTwoSeconds
+        } else {
+            self.testSession.timeRate = SKTestSession.TimeRate.monthlyRenewalEveryThirtySeconds
```

Comment:
Will this fail if we use `SKTestSession.TimeRate.monthlyRenewalEveryThirtySeconds` on iOS <16.4 and then sleep for 3 seconds later in the test?

Reply:
@fire-at-will  We only run backend tests on iOS 17 right now so... odds are this will never happen 😛

Reply:
🤪